.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@
 #
 # Apply all pre-commit hooks to all files:
 #   $ pre-commit run --all-files
-default_stages: [pre-commit]
+default_stages: [pre-push]
```

Comment:
Does this work as expected? I understand the intent, but please test it manually, as pre-commit configs often have unintuitive behavior.

Reply:
The check didn't allow me pushing [commit with an unused function](https://github.com/GoogleChromeLabs/chromium-bidi/pull/1421/commits/eeb80f58c0555b1aaff0b39343ac41c190b578c6). Eslint failed:
```
...
|eslint...................................................................Failed
- hook id: eslint
- exit code: 1

.../chromium-bidi/src/bidiServer/index.ts
  23:10  error    'foo' is defined but never used  @typescript-eslint/no-unused-vars
  23:16  warning  Unexpected empty function 'foo'  @typescript-eslint/no-empty-function

✖ 3 problems (1 error, 2 warnings)
```

Reply:
LGTM then